### PR TITLE
Update dropdown attribute

### PR DESCRIPTION
**Pull request by scand1n**

### CODE REVIEW

File: CloudFabric.EAV.Domain/Models/Attributes/ValueFromListAttributeConfiguration.cs [hunk 1]
```diff
@@ -34,6 +33,24 @@ public ValueFromListAttributeConfiguration(IEnumerable<IEvent> events) : base(ev
         public string? AttributeMachineNameToAffect { get; set; }
         public override EavAttributeType ValueType => EavAttributeType.ValueFromList;
 
+        public override List<string> Validate(AttributeInstance? instance)
+        {
+            var errors = base.Validate(instance);
+
+            if (instance == null)
```
Comment (dgon-jd) [inline CloudFabric.EAV.Domain/Models/Attributes/ValueFromListAttributeConfiguration.cs:40]: The 'null-check' is included in the `base.Validate()` and it throws error only if the `if (IsRequired && instance == null)` so there is no point in this here

**Review by dgon-jd — CHANGES_REQUESTED** [review]

**Review by ustims — APPROVED** [review]

File: CloudFabric.EAV.Tests/Tests.cs [hunk 2]
```diff
@@ -718,6 +719,266 @@ public async Task TestGetNumberAttribute_Success()
...
+
+        Func<Task> action = async () => await _eavService.CreateEntityConfiguration(entityConfigurationCreateRequest, CancellationToken.None);
+        await action.Should().ThrowAsync<Exception>();
+
+        valueFromListAttribute.ValuesList = new List<ValueFromListOptionCreateUpdateRequest>
+        {
+                new ValueFromListOptionCreateUpdateRequest(name: "Repeated Name", machineName: "firstTestOption", valueToAppend: 30),
+                new ValueFromListOptionCreateUpdateRequest(name: "Repeated Name", machineName: "secondTestOption", valueToAppend: null)
+        };
+
```
Comment (maxxsem) [inline CloudFabric.EAV.Tests/Tests.cs:851]: Do we need to call CreateEntityConfiguration() here again?

File: CloudFabric.EAV.Domain/Models/Attributes/ValueFromListAttributeConfiguration.cs [hunk 3]
```diff
@@ -34,6 +33,19 @@ public ValueFromListAttributeConfiguration(IEnumerable<IEvent> events) : base(ev
         public string? AttributeMachineNameToAffect { get; set; }
         public override EavAttributeType ValueType => EavAttributeType.ValueFromList;
 
+        public override List<string> Validate(AttributeInstance? instance)
+        {
+            var errors = base.Validate(instance);
+
+            if (instance is not ValueFromListAttributeInstance)
+            {
+                errors.Add("Cannot validate attribute. Expected attribute type: Dropdown");
```
Comment (maxxsem) [inline CloudFabric.EAV.Domain/Models/Attributes/ValueFromListAttributeConfiguration.cs:42]: Do we have a Dropdown type in our library?
Maybe it's more clear to use ValueFromList?

File: CloudFabric.EAV.Tests/Tests.cs (hunk 2 above)
Comment (scand1n) [reply CloudFabric.EAV.Tests/Tests.cs:851]: yes, it checks another case with repeated names

**Pull request merged**